File: app/src/main/java/com/example/evgeny/setlist_mobile/model/Set.java
package com.example.evgeny.setlist_mobile.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Evgeny on 23.07.2018.
 */

public class Set extends BaseModel {
    public String name;
    public String number;
    public String encore;
    public List<Song> songs;

    public Set() {
        songs = new ArrayList<>();
    }
}

File: app/src/main/java/com/example/evgeny/setlist_mobile/setlists/Set.kt
package com.example.evgeny.setlist_mobile.setlists

data class Set(val name: String, val encore: Int, val songs: List<Song> = arrayListOf()) : BaseModel(), SongListItem {

    override fun toString(): String {
        return "Set(name='$name', encore='$encore', songs=$songs)"
    }
}